release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -228,22 +225,24 @@ class RepoInfoExpansionState extends State<RepoInfoExpansion> {
                   0: FixedColumnWidth(240.0),
                 },
                 children: <TableRow>[
-                  for (String repoElement in widget.engineOrFramework == 'engine'
-                      ? ConductorStatus.engineRepoElements.values
-                      : ConductorStatus.frameworkRepoElements.values)
+                  // to do, navigate the map with enums
```

Comment:
TODOs should follow the Flutter styleguide and link to an issue

https://dart-lang.github.io/linter/lints/flutter_style_todos.html

Reply:
for context, we have tooling (such as the lint casey references) that does pattern matching on comment text to identify TODOs, which this would not match.

Reply:
Oops, my bad. Forgot to remove it. It was intended to help me drafting the PR.